Fix run-away AsciiDoc table in the new guides

### Before reporting an issue

- [X] I have searched existing issues
- [X] I have reproduced the issue with the [latest nightly release](https://github.com/keycloak/keycloak/releases/tag/nightly)

### Area

docs

### Describe the bug

The change in #22406 introduced a runaway table definition, which swallows and mis-aligns the content that follows. 

```
[ERROR] asciidoctor: ERROR: logging.adoc: line 808: table missing leading separator; recovering automatically
[ERROR] asciidoctor: ERROR: health.adoc: line 82: table missing leading separator; recovering automatically
```

### Version

main

### Expected behavior

Table should close with the same number of equal signs. Such errors should make the build fail

### Actual behavior

Runaway table, and errors in the log

### How to Reproduce?

Look at the build logs of the new guides

### Anything else?

I'll prepare a pull request for this.

--- a/docs/guides/pom.xml
+++ b/docs/guides/pom.xml
@@ -100,6 +100,11 @@
                         <sections>guides</sections>
                         <attribute-missing>warn</attribute-missing>
                     </attributes>
+                    <logHandler>
+                        <failIf>
+                            <severity>ERROR</severity>
+                        </failIf>
+                    </logHandler>
                 </configuration>
                 <executions>
                     <execution>

--- a/docs/guides/server/logging.adoc
+++ b/docs/guides/server/logging.adoc
@@ -104,7 +104,7 @@ The format string supports the symbols in the following table:
 |%t++{id}++|Thread ID|Render the thread ID.
 |%z{<zone name>}|Timezone|Set the time zone of log output to <zone name>.
 |%L|Line number|Render the line number of the log message.
-|====
+|===
 
 === Setting the logging format
 To set the logging format for a logged line, perform these steps:

